tests/ci/run_pytest.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+run_pytest.py is the script submitted to Azure ML that runs pytest.
+pytest runs all tests in the specified test folder unless parameters
+are set otherwise.
+"""
+
+import argparse
+import subprocess
+import logging
+import os
+
+from azureml.core import Run
+
+
+def create_arg_parser():
+    parser = argparse.ArgumentParser(description='Process inputs')
+    # test folder
+    parser.add_argument("--testfolder", "-f",
+                        action="store",
+                        default="./tests/unit",
+                        help="Folder where tests are located")
+    # internal test purposes
```

Comment:
this parameter is temporal and you plan to erase it?

Reply:
After this pr, I would like to finish adding the capability to take the unique pipeline number on DevOps and tag it in AML so there is a correlation between experiments on AML and pipeline runs.  That's a "nice to have" for others who clone this work. 

I removed the comment and kept the argument.